src/mumble/Settings.cpp

```diff
@@ -401,14 +401,8 @@ Settings::Settings() {
 	bJackStartServer  = false;
 	bJackAutoConnect  = true;
 
-#ifndef Q_OS_MAC
-	// Enable echo cancellation by default everywhere except for Macs as we currently
-	// on't support echo cancelling on Macs
-	bEcho = true;
-#else
 	bEcho = false;
-#endif
-	bEchoMulti = false;
+	iEchoOption = 0;  // ECHO_CANCEL_DISABLED = 0
```

Comment:
```suggestion
	iEchoOption = ECHO_CANCEL_DISABLED;
```

Reply:
Settings.cpp doesn't include the AudioInput.h.

Reply:
That would be an even better argument for defining the different values in a separate header file that can then be included on its own.